### PR TITLE
Add asyncquery callback endpoint

### PR DESCRIPTION
**Pull request by jdr0887**

### CODE REVIEW

File: node_normalizer/server.py [hunk 1]
```diff
@@ -72,42 +88,80 @@ async def shutdown_event():
...
     return response
 
 
+@app.post(
+    f"/asyncquery",
+    summary="Normalizes a TRAPI response object",
+    description="Returns the response object with a merged "
+    "knowledge graph and query graph bindings",
+)
+async def async_query(response: Response, callback: str):
```
Comment (cbizon) [inline node_normalizer/server.py:110]: I don't think this is quite right.  the callback is not a parameter, instead it is a field in response. So it'll be like:
```
{
   "callback": "http://example.com/whatever",
   "message": {...}
}

**Review by cbizon — CHANGES_REQUESTED** [review]
Other than the callback issue, looks good.   I have not come up with a clean way to test these async calls - do you have any ideas there?

File: node_normalizer/server.py (hunk 1 above)
Comment (PhillipsOwen) [reply node_normalizer/server.py:110]: it is possible to test the callback.

if working from hoe you can configure your home router to port forward to go to a debug session running in your local environment.

Comment (cbizon) [reply node_normalizer/server.py:110]: Yes, this is correct.  But is there a way to do it nicely in an automated way, e.g. as part of the travis-ci etc...

**Comment by jdr0887** [comment]
Reading the code in reasoner-pydantic, looks like there is a Query & AsyncQuery object that I think I should be using as the parameter for /query & /asyncquery respectively.  This would replace Response as the parameter.  Is that right?

https://github.com/TranslatorSRI/reasoner-pydantic/blob/1a78c6ad0a09bfee3bd389d9038a43f954bd957d/reasoner_pydantic/message.py#L99

https://github.com/TranslatorSRI/reasoner-pydantic/blob/1a78c6ad0a09bfee3bd389d9038a43f954bd957d/reasoner_pydantic/message.py#L119

**Comment by cbizon** [comment]
Yes, I think that's correct.  You can see how we did it in aragorn here: https://github.com/ranking-agent/aragorn/blob/main/src/robokop_app.py#L68

**Review by cbizon — APPROVED** [review]

**Pull request merged**